backend/router/data.py

```diff
@@ -12,49 +15,114 @@
 router = APIRouter()
 
 
-def cluster_data(confirmed):
-    df = pd.DataFrame(confirmed)
-
-    breaks = jenkspy.jenks_breaks(df["confirmed"], nb_class=CLUSTERS)
-
-    df["group"] = pd.cut(
-        df["confirmed"],
-        bins=breaks,
-        labels=CLUSTERS_LABELS,
-        include_lowest=True,
-    )
-    return df.to_dict("records")
+class DataScope:
+    ADM0 = "adm0"  # Base administrative area (countries)
+    ADM1 = "adm1"  # First level administrative area (states/provinces)
```

Comment:
I actually took this level-convention from mapbox (since is what we are using)
https://docs.mapbox.com/vector-tiles/reference/mapbox-boundaries-v3/#tilesets

If I'm not mistaken, `adm0`, which are the [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) territories, are the countries boundaries

Reply:
That's ok then! Good grace, I'm so glad I never worked with Mapbox before